Gemfile.lock

```diff
@@ -242,6 +246,7 @@ DEPENDENCIES
   pg (~> 1.1)
   puma (~> 5.0)
   rails (~> 7.0.4, >= 7.0.4.3)
+  rails-controller-testing
```

Comment:
- [ ] In your `db/migrate/` folder, you do not need the two migration file listed below because they are causing some error. 
   - **AddPostRefToLikes** - This ref to post has already been declared while you created the Likes table 👇🏾 
   
```ruby
class CreateLikes < ActiveRecord::Migration[7.0]
  def change
    create_table :likes do |t|
      t.references :post, foreign_key: true # Here the ref to post is already declared

      t.timestamps
    end
  end
end
```
   **_Error_**
   ![image](https://github.com/The-Normal-One1/Blog-App/assets/84156714/55f4a5af-5e06-493c-b303-9e731e5a0a71)

   - **RemovePostIdFromLikes**
   
   ![image](https://github.com/The-Normal-One1/Blog-App/assets/84156714/f74e84c1-b7f4-4d3e-89a8-fbf8b013ea5c)


Please remove this file.